glusterd2/volume/struct.go

```diff
@@ -41,6 +41,8 @@ const (
 	VkeyScrubFrequency = "bit-rot.scrub-freq"
 	// VkeyScrubThrottle is the key for controls scrubber throttle
 	VkeyScrubThrottle = "bit-rot.scrub-throttle"
+	//VkeyFeaturesGfproxyd is the key which enables or disables gfproxyd
+	VkeyFeaturesGfproxyd = "config.gfproxyd"
```

Comment:
Unrelated to your patch.

@prashanthpai, @kshlm what is the best way to maintain a feature is enabled or not in volinfo?